frontend/vite.config.ts

```diff
@@ -19,22 +19,4 @@ export default defineConfig({
     test: {
         include: ['src/**/*.{test,spec}.{js,ts}'],
     },
-    server: {
```

Comment:
can we please add this back? without this I can't run development locally unless I use docker for everything.

Reply:
maybe we could just have separate vite configs to give you what you want.